.gitignore

```diff
@@ -3,3 +3,4 @@ Cargo.lock
 *.log
 *.rs.rustfmt
 .gdb_history
+.idea/
```

Comment:
Ideally, this would live in your git user config and not the repository. Not a blocker IMO